metrics/src/vespa/metrics/common/memory_usage_metrics.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#include "memory_usage_metrics.h"
+#include <vespa/vespalib/util/memoryusage.h>
+
+namespace metrics {
+
+MemoryUsageMetrics::MemoryUsageMetrics(metrics::MetricSet* parent)
+    : MetricSet("memory_usage", {}, "The memory usage for a given component", parent),
+      _allocated_bytes("allocated_bytes", {}, "The number of allocated bytes", this),
+      _used_bytes("used_bytes", {}, "The number of used bytes (<= allocatedbytes)", this),
+      _dead_bytes("dead_bytes", {}, "The number of dead bytes (<= usedbytes)", this),
```

Comment:
Consider changing description to match metric name: "<= used_bytes"

Reply:
Done